doc/man/8/rbdmap.rst

```diff
@@ -0,0 +1,43 @@
+:orphan:
+
+=========================================
+ rbdmap -- map RBD devices at boot time
+=========================================
+
+.. program:: rbdmap
+
+Synopsis
+========
+
+| **rbdmap**
+
+
+Description
+===========
+
+**rbdmap** is a shell script that is intended to be run at boot time
+by the init system (sysvinit, upstart, systemd). It looks for an
```

Comment:
While it is true that it is usually run by init system, a user can also easily run it to achieve the same thing without having to have it run every time a computer starts -- that is why I originally kept it as a script in /usr/bin.

Alternatively, we could skip creating the man page altogether and move the script to /usr/lib/... (libexecdir). I'm not sure how useful the standalone script in bindir is and it depends mostly on that.